README.md

```diff
@@ -159,7 +138,7 @@ $ c++ main.cc `pkg-config --cflags --libs gtk+-3.0 webkit2gtk-4.0` -o webview-ex
 # MacOS
 $ c++ main.cc -std=c++11 -framework WebKit -o webview-example
 # Windows (x64)
-$ c++ main.cc -mwindows -L./dll/x64 -lwebview -lWebView2Loader -o webview-example.exe
+$ script/build.bat
```

Comment:
Maybe we should have a more complete explanation of building on Windows, since `script/build.bat` is designed for this repository by default. What do you think?

Reply:
I agree, but I'm struggling to find a good place for it... Should I make a special section called "building on windows"? Even go developers should run it to install their dependencies, so if I just place the info below each section, it might get a little redundant.

Reply:
@dandeto Yes, a new section for Windows would probably be a good idea, since it's more involved due to DLLs. If you'd like to finish up this PR and send another to cover that, we can leave this section the way you've changed it now in the meantime.